### DISCUSSION TITLE
Schema definition and example documentation

### DISCUSSION THREAD

**Discussion by colincameron** [laravie/parser#30] [Q&A]
Is the schema for specifying how an XML file is parsed defined anywhere? What are the available operators? I'm struggling to find any documentation beyond the readme file. For example, how do I read repeated tags into an array?

Any help or pointers would be greatly appreciated.

**Comment by cristiradic** [comment]
I used the tests to see what can be done, but they are incomplete.
It would be great to have some documentation.
